src/components/providers/ConnectRedirectProvider.tsx

```
@@ -3,6 +3,8 @@ import { useRouter } from 'next/navigation';
 import { toast } from 'react-toastify';
```

ESLint

[error] 'toast' is defined but never used.

(@typescript-eslint/no-unused-vars)